#goal definition
float32 move_x		# in meters
float32 move_y		# in meters
float32 rotation	# in rad
bool ignore_rotation #if true robot is not rotated after moving. if false robot is rotated "rotation" relative to the starting orientation
---
#result definition
bool goal_reached #true if goal is reached
---
#feedback
float32 forward_dist_x	# forward distance to goal in x
float32 forward_dist_y	# forward distance to goal in y
float32 rotation_dist	# rotationaldistance to goal